dpnp/dpnp_utils/dpnp_utils_reduction.py

```diff
@@ -29,9 +29,7 @@
 __all__ = ["dpnp_wrap_reduction_call"]
 
 
-def dpnp_wrap_reduction_call(
-    a, out, _reduction_fn, _get_res_dt_fn, *args, **kwargs
-):
+def dpnp_wrap_reduction_call(a, out, _reduction_fn, res_dt, **kwargs):
```

Comment:
It'd better to rename input array to `usm_a` to highlight that usm_ndarray is expecting:
```suggestion
def dpnp_wrap_reduction_call(usm_a, out, _reduction_fn, res_dt, **kwargs):
```